### ISSUE TITLE
TypeError preventing playback after bot moved channels

### ISSUE THREAD

**Issue by PersonMeetup** [Rapptz/discord.py#6219]
## Summary

When moving bots from one voice channel to another, any playback requests in the same command do not appear to work. Instead, a `NoneType` is brought up.

## Reproduction Steps

```py
@bot.command()
async def summon(ctx):
    channel = ctx.message.author.voice.channel
    voice = get(ctx.bot.voice_clients, guild=ctx.guild)
    if voice and voice.is_connected():
        if ctx.voice_client is not None or channel:
            await ctx.voice_client.move_to(channel)
    else:
        voice = await channel.connect()
        print(f"Connected to voice channel {channel}")

    source = discord.FFmpegPCMAudio("audio.mp3")     # Credit to Josh on the Discord
    voice.play(discord.PCMVolumeTransformer(source)) # for these two lines of code
    voice.source.volume = 0.5
```
From this code, repeat the following actions:

1. Enter voice channel and summon the bot using the command
2. Once the bot has finished audio playback, switch to another channel
3. Repeat the command

## Expected Results

Bot moves to the user's current voice channel and plays the MP3 file.

## Actual Results

While the bot enters the channel, no sound is played. The console returns with the following Traceback Error:
```Traceback (most recent call last):
  File "C:\Users\Person (Leslie)\AppData\Local\Programs\Python\Python37\lib\site-packages\discord\player.py", line 598, in run
    self._do_run()
  File "C:\Users\Person (Leslie)\AppData\Local\Programs\Python\Python37\lib\site-packages\discord\player.py", line 591, in _do_run
    play_audio(data, encode=not self.source.is_opus())
  File "C:\Users\Person (Leslie)\AppData\Local\Programs\Python\Python37\lib\site-packages\discord\voice_client.py", line 635, in send_audio_packet
    self.socket.sendto(packet, (self.endpoint_ip, self.voice_port))
TypeError: str, bytes or bytearray expected, not NoneType
```

## Intents

```py
intents = discord.Intents.default()
intents.members = True
```

## Checklist

- [x] I have searched the open issues for duplicates.
- [x] I have shown the entire traceback, if possible.
- [x] I have removed my token from display, if visible.
- [x] I have provided the intents that my bot is using.

## System Information

- Python v3.7.9-final
- discord.py v1.5.1-final
- aiohttp v3.6.3
- system info: Windows 10 10.0.19041

**Comment by Liquid-Dev0001** [comment]
Also got thet Error. Really annoying i guess there is no way of handling it

**Comment by Rapptz** [comment]
This is a duplicate of an already fixed issue in 1.6, #5953.
